agents/registry.py

```
@@ -0,0 +1,64 @@
+    from agents.team.tickerbot import TickerBot
+    from agents.team.tradedesk import TradeDeskAgent
+
+    # GeneralAgent must stay last: it is the fallback when no handle matches.
+    return [TradeDeskAgent(), ShoppingAgent(), ResearchAgent(), TickerBot(), GeneralAgent()]
+
+
+_TEAM: Optional[List[TeamMember]] = None
+
+
+def get_team() -> List[TeamMember]:
+    global _TEAM
```

Ruff (0.15.20)

[warning] Using the global statement to update `_TEAM` is discouraged

(PLW0603)